src/overlays/actors/ovl_En_Fall2/z_en_fall2.c

```diff
@@ -15,7 +15,14 @@ void EnFall2_Destroy(Actor* thisx, PlayState* play);
 void EnFall2_Update(Actor* thisx, PlayState* play);
 void EnFall2_Draw(Actor* thisx, PlayState* play);
 
-#if 0
+void func_80C1B8A4(EnFall2*, PlayState*);
+void func_80C1B9D4(EnFall2*, PlayState*);
```

Comment:
We like to have the variable names in declarations, even if it's not strictly required:

```suggestion
void func_80C1B8A4(EnFall2* this, PlayState* play);
void func_80C1B9D4(EnFall2* this, PlayState* play);
```